article/app/controllers/ArticleController.scala

```diff
@@ -39,7 +39,11 @@ class ArticleController extends Controller with RendersItemResponse with Logging
     val newBlocks = page.article.fields.blocks.takeWhile(block => s"block-${block.id}" != lastUpdateBlockId)
     val blocksHtml = views.html.liveblog.liveBlogBlocks(newBlocks, page.article, Edition(request).timezone)
     val timelineHtml = views.html.liveblog.keyEvents("", KeyEventData(newBlocks, Edition(request).timezone))
-    val allPagesJson = Seq("timeline" -> timelineHtml, "numNewBlocks" -> newBlocks.size)
+    val allPagesJson = Seq(
+      "timeline" -> timelineHtml,
+      "numNewBlocks" -> newBlocks.size,
+      "mostRecentBlockId" -> page.article.fields.blocks.headOption.map(block => s"block-${block.id}").getOrElse("block-9")
```

Comment:
just a block it definitely won't find in the list, that would only happen if a liveblog was somehow published without any blocks (which isn't possible).  If we're looking since a certain block and it doesn't exist, we just return them all.
Having said all that, maybe it could be clearer.  I'll take a look.